### PR TITLE
[TK-11]: build create account screen

### PR DESCRIPTION
**Pull request by mqayyum05**
## Jira: [TK-11 ](https://shop-go.atlassian.net/browse/TK-11?atlOrigin=eyJpIjoiZDYzNGNjODdlOTc5NDdjY2I3OWU1MjQyNzJlMjA0NzAiLCJwIjoiaiJ9)
**Description**:
- Built create account screen
- validation for email has been implemented
- navigation flow has been fixed
- auto layout has been implemented

**How to test it?**
- only connections have been made so far

## Screenshots&Gifs:
test screen

### CODE REVIEW

File: Vape-X/LoginPageViewController.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    @IBOutlet weak var email: UITextField!
+    @IBOutlet weak var phoneNumber: UITextField!
+    @IBOutlet weak var password: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    @IBAction func next(_ sender: Any) {
+    }
```
Comment (haroonmaqsoodch) [inline Vape-X/LoginPageViewController.swift:22]: change the names of these buttons to "nextButton" and "loginButton"

File: Vape-X/LoginPageViewController.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+//  LoginPageViewController.swift
+//  Vape-X
+//
+//  Created by Maha Qayyum on 9/22/22.
+//
+
+import UIKit
+
+class LoginPageViewController: UIViewController {
+
```
Comment (haroonmaqsoodch) [inline Vape-X/LoginPageViewController.swift:11]: Your screen is CreateAccount, change the name of the class

File: Vape-X/LoginPageViewController.swift [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    @IBOutlet weak var username: UITextField!
+    @IBOutlet weak var email: UITextField!
+    @IBOutlet weak var phoneNumber: UITextField!
+    @IBOutlet weak var password: UILabel!
```
Comment (haroonmaqsoodch) [inline Vape-X/LoginPageViewController.swift:15]: if it's a label then name it to "passwordLabel" or textfield at the end of their names

**Review by haroonmaqsoodch — COMMENTED** [review]
Your story number isn't TK-1 its TK-11 and it was a Create account screen, you need to fix the description

File: Vape-X/LoginPageViewController.swift (hunk 1 above)
Comment (mqayyum05) [reply Vape-X/LoginPageViewController.swift]: fixed

File: Vape-X/LoginPageViewController.swift (hunk 2 above)
Comment (mqayyum05) [reply Vape-X/LoginPageViewController.swift]: fixed

File: Vape-X/LoginPageViewController.swift (hunk 3 above)
Comment (mqayyum05) [reply Vape-X/LoginPageViewController.swift]: fixed

File: Vape-X/CreateAccountController.swift [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+//
+
+import UIKit
+
+class CreateAccountController: UIViewController {
+
+    @IBOutlet weak var usernameFeild: UITextField!
+    @IBOutlet weak var emailFeild: UITextField!
+    @IBOutlet weak var phoneNumberFeild: UITextField!
+    @IBOutlet weak var passwardLabel: UILabel!
```
Comment (ammadahmed22) [inline Vape-X/CreateAccountController.swift:15]: I saw you have a password label connection. You also have  3 other labels why is there no connection for them?

**Pull request merged**